tools/killswitch/src/errors.rs

```diff
@@ -1,9 +1,25 @@
 use nomad_base::ChainCommunicationError;
 use nomad_ethereum::EthereumSignersError;
+use rusoto_core::{credential::CredentialsError, RusotoError};
+use rusoto_s3::GetObjectError;
+use serde_yaml::Error as YamlError;
+use std::io::Error as IOError;
 
 /// `Error` for KillSwitch
 #[derive(Debug, thiserror::Error)]
 pub enum Error {
+    /// Cannot find local AWS credentials
+    #[error("BadCredentials: Cannot find AWS credentials: {0}")]
```

Comment:
do these messages add any error info that is not already contained in the inner error's message?

Reply:
I wanted these to have more context and err on the side of being too verbose